src/components/common/Avatar/Avatar.module.css

```
@@ -0,0 +1,7 @@
+.Avatar{
```

let's use lowercase for `className` to be consistent with other styles
```suggestion
.avatar{
```

---

The `img` tag is `inline` by default. All `inline` elements' height is calculated differently from the `block` ones. 
The `inline` element height consists of element height + line-height. To avoid any unwanted height discrepancies, let's make the image `block`.
```suggestion
.Avatar{
  display: block;
```